csphotoselector.css

```diff
@@ -278,3 +278,41 @@
 .CSPhoto_container_active {
   left: 11px;
 }
+
+/* responsive support */
+/* optimized for iPhone 5 and upper
+ * will produce a 1-column albums list
+ * and a 2-columns photos list
+ */
+@media screen and (max-width: 480px) {
```

Comment:
Can you change this to 650px. Right now it gets cut off a bit between that and 480.